linter/lintmain/lintmain.go

```diff
@@ -19,20 +21,37 @@ var config *Config
 func Run(cfg Config) {
 	config = &cfg // TODO(quasilyte): don't use global var for this
 	log.SetFlags(0)
-	cmdutil.DispatchCommand(subCommands)
-}
 
-var subCommands = []*cmdutil.SubCommand{
-	{
-		Main:  check.Main,
-		Name:  "check",
-		Short: "run linter over specified targets",
-	},
-	{
-		Main:  printVersion,
-		Name:  "version",
-		Short: "print linter version",
-	},
+	// makeExample replaces all ${linter} placeholders to a bound linter name.
+	makeExamples := func(examples ...string) []string {
+		from := "${linter} "
+		to := cfg.Name + " "
```

Comment:
Why extra spaces here and above?

Reply:
Now there is no `${linter}` at all, only `%s`.